.expeditor/verify.pipeline.yml

```diff
@@ -601,6 +602,7 @@ steps:
       - /tmp/event-service.log
       - /tmp/compliance.log
       - /tmp/nodemanager.log
```

Comment:
I take it this one doesn't actually exist?

Reply:
it does not